hugo/assets/scss/components/authors.scss

```diff
@@ -0,0 +1,190 @@
+@import '../config/colors';
+@import '../config/sizes';
+@import '../config/typography';
+@import '../mixins/screen';
+
+.authors {
+    $self: &;
+
+    --author-img-size: 24px;
+    --author-icon-color: #{ $c-blue };
+    --author-icon-size: 20px;
+    --author-popover-bg-color: #{ $c-blue--lightest };
+    --author-popover-border-color: #{ $c-grey-blue };
+    --author-text-color: #{ $c-blue };
+
+    align-items: center;
+    display: flex;
+    flex-wrap: wrap;
+    gap: 0.75rem $p-gutter;
+    position: relative;
+    width: fit-content;
+
+    &__item {
```

Comment:
I'm missing a hover/focus state on the item: Maybe change the color of the name or hover?